### PR TITLE
Develop djones errors

### PR DESCRIPTION
**Pull request by djones1040**
fixed a bug in DECam uploads and added some slightly more descriptive error messages for when things fail

### CODE REVIEW

**Comment by djones1040** [comment]
just added checking for multiple running instances of the most computationally-intensive crons - note that we now require the tendo module.

File: YSE_App/data_ingest/QUB_data.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
 from django.db.models import Q
 from YSE_App.models import Transient, TransientTag
 import sys
+from tendo import singleton
```
Comment (davecoulter) [inline YSE_App/data_ingest/QUB_data.py:28]: DJ -- did you add a new package to your local environment? I checked yse_virtual, and I don't see this dependency on prod. If this requires a new package, you'll also need to update the requirements.txt. Next week PM and I will be happy to help with that process.

**Review by davecoulter — CHANGES_REQUESTED** [review]
See my comment on `from tendo import singleton` in `QUB_data.py`

**Review by davecoulter — APPROVED** [review]
Looks good!

**Pull request merged**